android/modules/locale/src/java/ti/modules/titanium/locale/LocaleModule.java

```diff
@@ -51,6 +56,58 @@ public String getCurrentLocale()
 		return TiPlatformHelper.getInstance().getLocale();
 	}
 
+	@TargetApi(33)
+	@Kroll.method
+	@Kroll.setProperty
+	public void setApplicationLocales(String locales)
+	{
+		LocaleListCompat appLocale = LocaleListCompat.forLanguageTags(locales);
+		androidx.appcompat.app.AppCompatDelegate.setApplicationLocales(appLocale);
+	}
```

Comment:
```suggestion
	@Kroll.setProperty
	public void setApplicationLocales(String locales)
	{
		if (Build.VERSION.SDK_INT >= 33) {
			LocaleListCompat appLocale = LocaleListCompat.forLanguageTags(locales);
			AppCompatDelegate.setApplicationLocales(appLocale);
		} else {
			Log.w(TAG, "Only available for Android 33 and later");
		}
	}
```

Only use `@Kroll.setProperty` and add the SDK check in the method, so it will show a warning on older devices but is still available